src/gui/widgets/assembly.cc

```diff
@@ -652,10 +652,31 @@ void PCSX::Widgets::Assembly::draw(GUI* gui, psxRegisters* registers, Memory* me
                 }
                 ImGui::Text("  %s:%8.8x %c%c%c%c %8.8x: ", section, dispAddr, tc(b[0]), tc(b[1]), tc(b[2]), tc(b[3]),
                             code);
+                auto toggleBP = [&]() mutable {
```

Comment:
`[&]` or `[=]` are deprecated captures starting in C++20, and we should explicitly capture what we need instead (which, imho, is a bit idiotic, but might as well try and write code that won't break in the next few C++ versions).

Also `mutable` shouldn't be required here? It's only if you're mutating a local capture, like the following:

```c++
std::function<unsigned()> getCounter() {
  unsigned counter = 0;
  return [counter]() mutable -> unsigned { return counter++; };
}
```